src/status_im/communities/core.cljs

```diff
@@ -175,11 +185,38 @@
   {:json-rpc/call [{:method      "wakuext_requestToJoinCommunity"
                     :params      [{:communityId community-id}]
                     :js-response true
-                    :on-success  #(re-frame/dispatch [::requested-to-join %])
+                    :on-success  #(do
+                                    (re-frame/dispatch [::requested-to-join %])
+                                    (re-frame/dispatch [:communities/get-user-requests-to-join]))
                     :on-error    #(do
                                     (log/error "failed to request to join community" community-id %)
                                     (re-frame/dispatch [::failed-to-request-to-join %]))}]})
 
+
+
+(rf/defn get-user-requests-to-join
+  {:events [:communities/get-user-requests-to-join]}
+  [cofx]
```

Comment:
`cofx` is not being used. Hey, this is a common example for why some macros can be problematic. The linter doesn't understand `cofx` is not used, nor does the Clojure language server.

Reply:
will remove